### PR TITLE
Update BEFH-BETAFPVG473

### PR DESCRIPTION
**Pull request by liyouluo**
Hi, please review the target update.

Thank you.

### CODE REVIEW

**Comment by sugaarK** [comment]
You need to make a config target. 4.5 does not use the unified target branch and the unified farther branch is about to be locked. We have no interest in back porting new hardware to 4.4

**Review by nerdCopter — DISMISSED** [review]

**Comment by nerdCopter** [comment]
documentation: https://betaflight.com/docs/development/manufacturer/creating-configuration
repository: https://github.com/betaflight/config

File: configs/default/BEFH-BETAFPVG473.config [hunk 1]
```diff
@@ -105,7 +117,7 @@ set serialrx_provider = CRSF
 set blackbox_device = SPIFLASH
 set dshot_idle_value = 450
 set dshot_burst = OFF
-set dshot_bitbang = OFF
+set dshot_bitbang = AUTO
```
Comment (haslinghuis) [inline configs/default/BEFH-BETAFPVG473.config:120]: @liyouluo what is the reason for this change? Asking because we had an issue which would be solved in 4.5.1. Have you checked the board with 4.6 development firmware?

**Comment by liyouluo** [comment]
> You need to make a config target. 4.5 does not use the unified target branch and the unified farther branch is about to be locked.

Thanks for the reminder, there is already a BEFH-BETAFPV G473 target in betaflight/config, I will submit a PR to update the target.

**Comment by liyouluo** [comment]
> documentation: https://betaflight.com/docs/development/manufacturer/creating-configuration repository: https://github.com/betaflight/config

OK. I will submit a PR to update the target.

File: configs/default/BEFH-BETAFPVG473.config (hunk 1 above)
Comment (liyouluo) [reply configs/default/BEFH-BETAFPVG473.config:120]: > @liyouluo what is the reason for this change? Asking because we had an issue which would be solved in 4.5.1. Have you checked the board with 4.6 development firmware?

Turning on dshot_bidir is the best setting because it can use RPM filter. I did some tests last night, PC13 (MOTOR 4) uses TIM8_CH4N, MOTOR 4 cannot implement dshot_bidir when dshot_bitbang = OFF. This problem exists in 4.4.2-4.6.0, and firmware earlier than 4.4.2 has not been tested.

**Review by haslinghuis — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]
